api/app/signals/apps/signals/models/signal.py

```diff
@@ -161,6 +161,9 @@ def allows_contact(self) -> bool:
         based on the latest filled in feedback object
         If no feedback object exists allowing contact is True
         """
+        if not settings.FEATURE_FLAGS.get('REPORTER_MAIL_DISABLE_CONTACT_FEEDBACK_ALLOWS_CONTACT', True):
```

Comment:
See comment on `settings/base.py`. Rename feature flag to `REPORTER_MAIL_CONTACT_FEEDBACK_ALLOWS_CONTACT_ENABLED`